samples/sensor/mhz19b/src/main.c

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2021 G-Technologies Sdn. Bhd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <sys/printk.h>
+#include <drivers/sensor.h>
+#include <drivers/sensor/mhz19b.h>
+
+void main(void)
+{
+	const struct device *dev;
+	struct sensor_value val;
+	int ret;
+
+	printk("Winsen MH-Z19B CO2 sensor application\n");
+
+	dev = DEVICE_DT_GET(DT_INST(0, winsen_mhz19b));
```

Comment:
Please don't use DT_INST in sample code; it's for drivers. Try DEVICE_DT_GET_ANY or DEVICE_DT_GET_ONE instead.

Reply:
Done